pystapi-client/README.md

```diff
@@ -0,0 +1,53 @@
+# pystapi-client
+
+PySTAPI Client is a Python library for interacting with [STAPI](https://github.com/stapi-spec/stapi-spec) endpoints. Below is a overview of the supported endpoints and examples.
+
+## Installation
+
+Install from PyPi.
+The dependencies for **pystapi-client** are the Python [httpx](https://www.python-httpx.org/) and [dateutil](https://dateutil.readthedocs.io) libraries.
+
+```shell
+python -m pip install pystapi-client
+```
+
+## Currently Supported Endpoints
+
+These endpoints are fully implemented and available in the current version of PySTAPI Client.
+
+| Category | Endpoint | Description |
+|----------|----------|-------------|
+| Root | `/` | Root endpoint (for links and conformance) |
+| Root | `/conformance` | Conformance information |
+| Products | `/products` | List all products |
+| Products | `/products/{product_id}` | Get specific product |
+| Orders | `/orders` | List all orders |
+| Orders | `/orders/{order_id}` | Get specific order |
+
```

Comment:
I'm a little wary about including this table, as we'll have to keep it continually updated. I think I'd prefer to just point to the API docs, which will be generated from the code?